Add Manibus keyboard and layout from Blank Tehnologii (#11832)
qmk/qmk_firmware: keyboards/blank_tehnologii/manibus/readme.md

# Manibus keyboard

![Imgur](https://i.imgur.com/0l17Xyyl.jpg)

A 5x7 split ergonomic keyboard designed and sold by [Blank Tehnologii](https://blank.computer/).

* Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
* Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)

> *  Hardware Availability: [Blank Tehnologii](https://blank.computer/)

Reviewer: ```suggestion
* Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```

# Manibus keyboard

> ![Imgur](https://i.imgur.com/0l17Xyy.jpg)

Reviewer: ```suggestion
![Imgur](https://i.imgur.com/0l17Xyyl.jpg)
```

A 5x7 split ergonomic keyboard designed and sold by [Blank Tehnologii](https://blank.computer/).

> Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
> 
> Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
> 
> Hardware Availability: [Blank Tehnologii](https://blank.computer/)

Reviewer: ```suggestion
* Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
* Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
*  Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```

Make example for this keyboard (after setting up your build environment):

> ```make blank_tehnologii/manibus:default```

Reviewer: ```suggestion
    make blank_tehnologii/manibus:default
```

Example of flashing this keyboard:

> Left half: 
> ```make blank_tehnologii/manibus:default:dfu-split-left```
> 
> Right half:
> ```make blank_tehnologii/manibus:default:dfu-split-right```

Reviewer: ```suggestion
    make blank_tehnologii/manibus:default:dfu-split-left # Left half
    make blank_tehnologii/manibus:default:dfu-split-right # Right half
```

> See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started?id=_5-configure-your-build-environment-optional) and [handedness by EEPROM](https://docs.qmk.fm/#/feature_split_keyboard?id=handedness-by-eeprom) for more information.
> 

Reviewer: ```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
